Modules/TPC/src/Calibration.cxx

```diff
@@ -0,0 +1,89 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   PostProcessingInterface.cxx
+/// \author Thomas Klemenz
+///
+
+// O2 includes
+#include "TPC/Calibration.h"
+#include "TPCBase/Painter.h"
+#include "TPCBase/CDBInterface.h"
+
+// QC includes
+#include "QualityControl/QcInfoLogger.h"
+
+//root includes
+#include "TCanvas.h"
+
+using namespace o2::quality_control::postprocessing;
+
+namespace o2::quality_control_modules::tpc
+{
+
+void Calibration::initialize(Trigger, framework::ServiceRegistry&)
```

Comment:
See comment in 
`void Clusters::initialize(o2::framework::InitContext& /*ctx*/)`